sql/maintenance_utils.sql

```diff
@@ -21,7 +21,8 @@ CREATE OR REPLACE FUNCTION move_chunk(
 
 CREATE OR REPLACE FUNCTION compress_chunk(
     uncompressed_chunk REGCLASS,
-    if_not_compressed BOOLEAN = false
+    if_not_compressed BOOLEAN = false,
+    ignore_errors BOOLEAN = false
 ) RETURNS REGCLASS AS '@MODULE_PATHNAME@', 'ts_compress_chunk' LANGUAGE C STRICT VOLATILE;
```

Comment:
Since you are changing the signature of the `compress_chunk` function here, you need to add it to `latest-dev.sql` as well and drop *all* overloads of the `compress_chunk` function.